packages/clerk-js/src/utils/captcha.ts

```diff
@@ -99,7 +99,7 @@ export const getCaptchaToken = async (captchaOptions: { siteKey: string; scriptU
              * By setting retry to 'never' the responsibility for implementing retrying is ours
              * https://developers.cloudflare.com/turnstile/reference/client-side-errors/#retrying
              */
-            if (retries < 2 && shouldRetryTurnstileErrorCode(errorCode)) {
+            if (retries < 2 && shouldRetryTurnstileErrorCode(errorCode + '')) {
```

Comment:
One of these would be more clear:

```suggestion
            if (retries < 2 && shouldRetryTurnstileErrorCode(errorCode.toString())) {
```

```suggestion
            if (retries < 2 && shouldRetryTurnstileErrorCode(String(errorCode))) {
```

Reply:
Updated!